doc/api/test.md

```diff
@@ -1521,6 +1521,24 @@ Emitted when all subtests have completed for a given test.
 
 Emitted when a test starts.
 
+### Event: `'test:stderr'`
+
+* `data` {Object}
+  * `file` {string} The path of the test file.
```

Comment:
For the `test:diagnostic` event, this can also be `undefined`. Is that possible for these new events?

Reply:
no. `test:diagnostic` can be produced inside the REPL. these new events are only useful with `--test`